modules/Kconfig.mcuboot

```diff
@@ -254,6 +254,21 @@ config MCUBOOT_BOOTLOADER_MODE_FIRMWARE_UPDATER
 	  a dedicated firmware updater application used to update the slot0_partition
 	  application.
 
+config MCUBOOT_BOOTLOADER_MODE_SINGLE_APP_RAM_LOAD
```

Comment:
should also be added to `share/sysbuild` so that it can be selected from a sysbuild project and propagated to MCUboot and application, can either add that here or I can add it in a separate PR

Reply:
I gave it a shot, hopefully is right.